src/test/scala/tech/cryptonomic/conseil/tezos/PlatformDiscoveryOperationsTest.scala

```diff
@@ -163,4 +171,83 @@ class PlatformDiscoveryOperationsTest
     }
   }
 
+  "listAttributeValues" should {
+
+    "return list of values of kind attribute of Fees without filter" in {
+      val avgFee = AverageFees(1, 3, 5, Timestamp.valueOf(LocalDateTime.of(2018, 11, 22, 12, 30)), "example1")
+
+      Await.result(dbHandler.run(TezosDatabaseOperations.writeFees(List(avgFee))), Duration.Inf)
```

Comment:
Waiting an infinite amount of time doesn't seem useful, and may create issues on a CI environment.

Why not defining a reasonably low expectation instead, so that a problematic test will fail faster?

Side note: if you prefer there's also an extension method `future.isReadyWithin(Span)`that returns a Boolean, without using `Await.ready`

Reply:
yes, you're right, changed